instfiles/xrdp-sesman.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=xrdp session manager
+Documentation=man:xrdp-sesman(8) man:sesman.ini(5)
+After=network.target
+StopWhenUnneeded=true
+BindsTo=xrdp.service
+
+[Service]
+Type=forking
+PIDFile=/var/run/xrdp-sesman.pid
```

Comment:
hardcode

Reply:
I've removed the file as its get build at compile time